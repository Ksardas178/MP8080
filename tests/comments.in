/*Comment 1*/
MOV A B
/*Comment 2*/
MOV B C
//
		/**/
/***
MOV 2 3/
*/////
MOV 3 4
//***********/
HLT