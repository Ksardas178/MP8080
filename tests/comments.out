**
[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: 01111000
00000001: 01000001
00000010: 01011100
00000011: 01110110
